model/openapi.yml

```diff
@@ -549,7 +549,14 @@ components:
       # unenroll actions have no `data` attribute
     actionRequestDiagnostics:
       description: The REQUEST_DIAGNOSTICS action data.
-      # diagnostics actions have no `data` attribute
+      properties:
```

Comment:
Just to double check, this is backwards compatible with older agents that don't understand this yet? They'll just ignore it?